Docs/source/pages/commands.md

```diff
@@ -1782,6 +1782,79 @@ Simulates device rotation action in your game.
 
 ```
 
+#### ResetInput
+
+Clear all active input actions simulated by AltTester.
```

Comment:
should be `Clears` instead of `Clear`

Reply:
I will make a separate ticket for this typo not to block the merge of this PR :D